meta/autoid/autoid.go

```diff
@@ -529,7 +795,25 @@ func generateAutoIDByAllocType(m *meta.Meta, dbID, tableID, step int64, allocTyp
 		return m.GenAutoTableID(dbID, tableID, step)
 	case AutoRandomType:
 		return m.GenAutoRandomID(dbID, tableID, step)
+	case SequenceType:
+		return m.GenSequenceValue(dbID, tableID, step)
 	default:
 		return 0, errInvalidAllocatorType.GenWithStackByArgs()
 	}
 }
+
+func setSequenceBaseValue(m *meta.Meta, dbID, tableID, cycleValue int64) error {
+	return m.SetSequenceValue(dbID, tableID, cycleValue)
+}
+
+// setSequenceCycleRound is used to store the flag `round` which indicates whether the sequence is already in cycle.
+// round > 0 means the sequence is already in cycle, so the offset should be minvalue / maxvalue rather than sequence.start.
+// TiDB is a stateless node, it should know whether the sequence is already in cycle when restart.
+func setSequenceCycleRound(m *meta.Meta, dbID, tableID, round int64) error {
+	return m.SetSequenceCycle(dbID, tableID, round)
+}
+
+// getSequenceCycleRound is used to get the flag `round`, which indicates whether the sequence is already in cycle.
+func getSequenceCycleRound(m *meta.Meta, dbID, tableID int64) (int64, error) {
+	return m.GetSequenceCycle(dbID, tableID)
+}
```

Comment:
Why not calling `GetSequenceCycle` directly in the usage of `getSequenceCycleRound`? How can `getSequenceCycleRound` evolve in the future?

Reply:
done